crowdin.yml

```diff
@@ -32,6 +32,15 @@ files:
         pt-BR: pt-br
         zh-CN: zh-cn
         zh-TW: zh-tw
+  - source: /apps/site/snippets/en/**/*.bash
```

Comment:
is it dangerous to translate that ?

Reply:
These are never executed... Although the extension is .bash they are pretty much just text...

Reply:
Also they don't have chmod +x by default

Reply:
I mean dangerous like they can be break in the "past" we just translate the comment not the whole script

Reply:
so my question is, do we need to translate theses file ?

Reply:
Yes, we do.

> I mean dangerous like they can be break in the "past" we just translate the comment not the whole script

I don't think it is dangerous, the issue with MDX was that MDX uses a custom engine and has special things. This is plain text, no way for mistakes to happen.